.github/workflows/wait-and-upload.yml

```diff
@@ -137,42 +133,14 @@ jobs:
         with:
           context: .
           push: true
-          tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ inputs.platform }}.${{ inputs.compiler-nix-name }}${{ inputs.target-platform }}${{ (inputs.iog-full && '-iog-full') || (inputs.iog && '-iog') }}
+          tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ inputs.platform }}.${{ inputs.compiler-nix-name }}${{ inputs.target-platform }}${{ inputs.variant }}${{ inputs.iog }}
           build-args: |
             PLATFORM=${{ inputs.platform }}
             TARGET_PLATFORM=${{ inputs.target-platform }}
             COMPILER_NIX_NAME=${{ inputs.compiler-nix-name }}
-            MINIMAL=${{ inputs.minimal }}
+            VARIANT=${{ inputs.variant }}
             IOG=${{ inputs.iog }}
-            IOG_FULL=${{ inputs.iog-full }}
-
-  codespace-download:
-    env:
-      IMAGE_NAME: input-output-hk/devx-devcontainer
-    needs: codespace-upload
-    permissions:
-      packages: write
-    runs-on: ubuntu-latest
-    # We want a GitHub Codespace image for each combination of devx developer shell option.
-    # But, since the purpose of GitHub Codespace is to serve a complete development environment,
-    # the user is likely to always expect HLS (I don't see the point otherwise).
-    # Therefore, it doesn't seem useful to build an image on the `-minimal` flavor (without HLS),
-    # or the `-static` one (especially since the latter currently requires `-minimal` to work).
-    # Likely, we consider using `-iog` as the default and do not generate other images.
-    # Then the user choices left would be between native, `-windows` or `-js` target platforms,
-    # and the GHC version (currently `ghc810` and `ghc96`).
-    if: ${{ contains(fromJSON('["x86_64-linux", "aarch64-linux"]'), inputs.platform) && contains(fromJson('["","-windows","-js"]'), inputs.target-platform) && contains(fromJson('["ghc810","ghc96"]'), inputs.compiler-nix-name) && !inputs.minimal && inputs.iog }}
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Log in to the Container registry
-        uses: docker/login-action@v2.1.0
-        with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why can we drop all of this now?

Reply:
I realise there was no point in doing it in a separate job, and doing it in the same job will speed up the whole process (as it reuse the Docker layer cache) will still harnessing that the `devx-container` still works as expected!